.bcr/presubmit.yml

```diff
@@ -0,0 +1,10 @@
+bcr_test_module:
+  module_path: "e2e/smoke"
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+  tasks:
+    run_tests:
+      name: "Run test module"
+      platform: ${{ platform }}
+      test_targets:
+        - "//..."
```

Comment:
Just to confirm, this would run `bazel test ...` from the ruleset root?

Reply:
no, it runs it under the `module_path` from line 2 - which I should change to `examples/gem` if we stick with the layout of this PR.

Note, however, that the BCR presubmit runs against the published release artifact, so whatever we use here for testing can't be omitted from that artifact.